### PR TITLE
Qr code support page

### PR DESCRIPTION
**Pull request by omartehsin1**
# Summary | Résumé

Added the  QR Support page for both English and French. 

Only the "Contact Us" links direct users to the "Contact Us" page, the rest of the links are blank.

Link: https://covid-alert-qr-code-sup-wb4ebv.herokuapp.com/

Trello: https://trello.com/c/n6bZOvt2/668-support-for-registering-qr-codes-page-documentation-for-website

### CODE REVIEW

File: locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -2251,6 +2252,139 @@ msgstr ""
 msgid "People in your province or territory are not yet able to scan a QR code for COVID Alert.You can still make a poster with your code.  That way, you’ll be ready when people are able to scan these codes."
 msgstr ""
 
+#: register/templates/register/qr_support_page.html
+#, fuzzy
+#| msgid "Make another poster"
+msgid "Help with your poster"
+msgstr "Créer une autre affiche"
```
Comment (dsamojlenko) [inline locale/fr/LC_MESSAGES/django.po:2259]: This fuzzy match needs to be resolved

File: locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -2251,6 +2252,139 @@ msgstr ""
...
+
+#: register/templates/register/qr_support_page.html
+msgid "Yukon"
+msgstr "Yukon"
+
+#: register/templates/register/qr_support_page.html
+#, fuzzy
+#| msgid "For any other help with the portal"
+msgid "For help with the app"
+msgstr "Pour toute autre assistance avec le portail"
```
Comment (dsamojlenko) [inline locale/fr/LC_MESSAGES/django.po:2362]: This fuzzy match should be resolved

File: locale/fr/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -2336,12 +2470,21 @@ msgid "Start now"
...
-#, python-format
+#, fuzzy, python-format
+#| msgid ""
+#| "\n"
+#| "  <h2> Questions? </h2>\n"
+#| "    <p>\n"
+#| "      Visit <a href=\"\">Help with your poster</a><br>\n"
+#| "      Or <a href =\"%(link)s\">Contact us</a>\n"
+#| "   </p>\n"
+#| "   "
```
Comment (dsamojlenko) [inline locale/fr/LC_MESSAGES/django.po:2481]: This fuzzy match should be resolved

**Comment by YedidaZalik** [comment]
Please change the second last heading to "For more help" (currently says "For help with the app" which should be the last heading only.
In the French, please change the headings to French (currently in English).
thanks!

**Review by dsamojlenko — APPROVED** [review]

**Pull request merged**